ibm/mas_devops/roles/dro/README.md

```diff
@@ -0,0 +1,103 @@
+dro
```

Comment:
As this is a new role then we need to update https://github.com/ibm-mas/ansible-devops/blob/master/mkdocs.yml to contain it.

Also if DRO will only work with certain MAS versions then it should be state in the docs.

Reply:
@yuvraj-vansure @lokesh-sreedhara this question is still outstanding: 
> Also if DRO will only work with certain MAS versions then it should be state in the docs.

I believe we are backporting changes into certain levels of MAS, so if a user was to try to install DRO but had version 8.10.1 of MAS then I don't believe it would work?

Reply:
@whitfiea jus updated the readme with the supported MAS versions.